### DISCUSSION TITLE
Unable to login with a different email

### DISCUSSION THREAD

**Discussion by chsunny548** [community/community#36545] [Other Feature Feedback, Questions, & Ideas]
Hi 

Whenever I INSTALL THE GITHUB APP AND OPEN IT IT AUTO LOGINS TO ONE OF MY OLD EMAILS. THERE IS NO OPTION TO LOGIN WITH A DIFFERENT EMAIL

**Comment by Nishu0** [comment]
GitHub account credentials are stored under Control Panel → User Accounts → Credential Manager → Manage Windows Credentials.  
@chsunny548 
To change your account using different emails do the following steps

Steps:
1. Open the Control Panel, go to User Accounts, then select Credential Manager. Alternatively, use the search box on the taskbar to search for ‘credential manager.’ 
2. You will see Git credentials in the list (e.g. git:https://). Click on it, remove your account.  
  
![](https://i.stack.imgur.com/9K7sE.png)  
3. Run 'git push' it will ask for your GitHub credentials, enter your new credentials.

Hope you are able to change your account.
If you have any doubt regarding this, feel free to ask.

**Comment by Nishu0** [comment]
@chsunny548  please mark this as an answer if you have no further queries so that others can know that the question has been answered.

**Comment by ossanna16** [comment]
Hey there @chsunny548 and thank you for asking a good question! I'm glad you found our community 🙂
